src/components/OptionsList.js

```diff
@@ -82,8 +68,23 @@ const OptionsList = ({
     canSelectMultipleOptions,
     hideSectionHeaders,
     onSelectRow,
+    headerMessage,
+    headerTitle,
 }) => (
     <View style={[styles.flex1]}>
+        {headerMessage ? (
```

Comment:
Yes, but not a super obvious one. Anytime we are dealing with strings and using the short circuit style ternary in JSX we need to be careful since an empty string can still be passed to a `Text` component. This is one of those weird differences between React Native and regular React on the web but basically throws nasty errors on mobile and breaks the build. So I think we should stop using `&&` (at least for strings). We might be fine in this example since I think it's OK to render an empty string inside a `View`. But e.g. where we use `headerTitle` we would not want to do `&&`. 

https://github.com/Expensify/Expensify.cash/blob/master/STYLE.md#important-note